### ISSUE TITLE
Occasional error when starting PyCBC Live

### ISSUE THREAD

**Issue by titodalcanton** [gwastro/pycbc#2554]
Sometimes when PyCBC Live starts, it hits the following error right after generating the templates. Possibly because one of the initially-present frame files has rotated out of existence in the mean time.
```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/lib64/python2.7/runpy.py", line 240, in run_path
    return _run_module_code(code, init_globals, run_name, path_name)
  File "/usr/lib64/python2.7/runpy.py", line 82, in _run_module_code
    mod_name, mod_fname, mod_loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/bin/pycbc_live", line 516, in <module>
    dq_padding=args.data_quality_padding)
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib/python2.7/site-packages/pycbc/strain/strain.py", line 1279, in __init__
    increment_update_cache=increment_update_cache)
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib/python2.7/site-packages/pycbc/frame/frame.py", line 472, in __init__
    self.update_cache()
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib/python2.7/site-packages/pycbc/frame/frame.py", line 486, in update_cache
    cache = locations_to_cache(self.frame_src, latest=True)
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib/python2.7/site-packages/pycbc/frame/frame.py", line 118, in locations_to_cache
    flist = [max(flist, key=os.path.getctime)]
  File "/home/pycbc.live/production/O3/pycbc_live_O3_production_env/lib64/python2.7/genericpath.py", line 64, in getctime
    return os.stat(filename).st_ctime
OSError: [Errno 2] No such file or directory: '/dev/shm/kafka/V1/V-V1_llhoft-1235938558-1.gwf'
```

**Comment by titodalcanton** [comment]
I *suspect* the problem is that sometimes a frame file disappears between the `glob()` and `getctime()` calls. That code *should* be fast and admittely this scenario seems unlikely, but I see no other explanation. I pushed a fix in #2564 but I am not sure how to test it other than wait and see…

**Comment by titodalcanton** [comment]
I have not seen this error in the last ~week of the replay analysis, which restarted itself ~10 times over that period. I am closing this for now.
